.github/workflows/main.yml

```diff
@@ -113,7 +113,7 @@ jobs:
     - name: Create k3d registry
       run: k3d registry create "${REG_BASE}" --port "${REG_PORT}"
     - name: Create k3d cluster
-      run: k3d cluster create --wait --registry-use "${REGISTRY}"
+      run: k3d cluster create --wait --image docker.io/rancher/k3s:v1.21.5-k3s1 --registry-use "${REGISTRY}"
```

Comment:
Instead of hardcoding to one particular patch version, would it make more sense to lock on to a Minor version?

Something like
```
IMAGE_TAG=$(curl --silent "https://update.k3s.io/v1-release/channels/v1.21" | egrep -o '/v[^ ]+"' | sed -E 's/\/|\"//g' | sed -E 's/\+/\-/')
k3d cluster create --wait --image docker.io/rancher/k3s:${IMAGE_TAG} --registry-use "${REGISTRY}"
```

Reply:
Sure, let's follow up with something like that in a follow-up pr.